spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -277,6 +277,19 @@ def delete_beta_tester_from_beta_groups(beta_tester_id: nil, beta_group_ids: [])
           test_flight_request_client.delete("betaTesters/#{beta_tester_id}/relationships/betaGroups", nil, body)
         end
 
+        def delete_beta_testers_from_app(beta_tester_ids: [], app_id: nil)
```

Comment:
The title of the PR is "add and delete" but this is named `delete_beta_testers_from_app` 🤔 Should this method actually be called something like `update_beta_testers_from_app`?

Reply:
It's just that you can delete more than one tester at a time, which is very useful to me. My English is not very good. I'm sorry.

Reply:
Delete multiple testers, which greatly improves efficiency.